tests/transports-governance/test_helpers.py

```
@@ -0,0 +1,644 @@
+        print(f"  Average duration: {stats['avg_duration']:.3f}s")
+        print(f"  Min duration: {stats['min_duration']:.3f}s")
+        print(f"  Max duration: {stats['max_duration']:.3f}s")
+        print(f"  Total duration: {stats['total_duration']:.3f}s")
+
+
+class ChatCompletionHelper:
+    """Helper for chat completion testing"""
+
+    @staticmethod
+    def generate_test_messages(
+        complexity: str = "simple", token_count_estimate: int = None
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)